base_field_validator/demo/ir_model_demo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="regex_mail" model="ir.model.fields.regex">
+            <field name="regex">[^@]+@[^@]+\.[^@]+</field>
+            <field name="name">Email</field>
```

Comment:
Maybe this could go into data instead of demo. I'd like to know what others think about it.